### PR TITLE
Restructure some of documentation comments

### PR DESCRIPTION
**Pull request by daschult**
Resolves https://github.com/Azure/ms-rest-js/issues/20.

### CODE REVIEW

File: lib/credentials/tokenCredentials.ts [hunk 1]
```diff
@@ -9,17 +9,17 @@ import { ServiceClientCredentials } from "./serviceClientCredentials";
...
 const DEFAULT_AUTHORIZATION_SCHEME = "Bearer";
 
-/**
- * Creates a new TokenCredentials object.
- *
- * @constructor
- * @param {string} token               The token.
- * @param {string} authorizationScheme The authorization scheme.
- */
 export class TokenCredentials implements ServiceClientCredentials {
```
Comment (RikkiGibson) [inline lib/credentials/tokenCredentials.ts]: no doc for the class itself?

File: lib/webResource.ts [hunk 2]
```diff
@@ -298,65 +298,90 @@ export class WebResource {
...
- * @param {object|string|boolean|array|number|null|undefined} [options.body] - The request body. It can be of any type. This method will JSON.stringify() the request body.
- * @param {object} [options.serializationMapper] - Provides information on how to serialize the request body.
- * @param {object} [options.mappers] - A dictionary of mappers that may be used while [de]serialization.
- * @param {object} [options.deserializationMapper] - Provides information on how to deserialize the response body.
- * @param {boolean} [options.disableJsonStringifyOnBody] - Indicates whether this method should JSON.stringify() the request body. Default value: false.
- * @param {boolean} [options.bodyIsStream] - Indicates whether the request body is a stream (useful for file upload scenarios).
- * @returns {object} WebResource Returns the prepared WebResource (HTTP Request) object that needs to be given to the request pipeline.
- */
 export interface RequestPrepareOptions {
+  /**
```
Comment (RikkiGibson) [inline lib/webResource.ts]: Very good

File: lib/webResource.ts [hunk 3]
```diff
@@ -298,65 +298,90 @@ export class WebResource {
...
+   *  - "accept-language" by default is set to "en-US"
+   *  - "x-ms-client-request-id" by default is set to a new Guid. To not generate a guid for the request, please set options.disableClientRequestId to true
+   */
   headers?: { [key: string]: any };
+  /**
+   * When set to true, instructs the client to not set "x-ms-client-request-id" header to a new Guid().
+   */
   disableClientRequestId?: boolean;
+  /**
+   * The request body. It can be of any type. This method will JSON.stringify() the request body.
```
Comment (RikkiGibson) [inline lib/webResource.ts]: Can we even succinctly describe what we will do to the body any more? Certainly no references to "this method" as there is no method being described by this doc comment..

File: lib/webResource.ts (hunk 2 above)
Comment (daschult) [reply lib/webResource.ts]: Added in https://github.com/Azure/ms-rest-js/pull/166/commits/6898d391c4fc522faea07c70a358da2867a99791.

File: lib/webResource.ts (hunk 3 above)
Comment (daschult) [reply lib/webResource.ts]: Done in https://github.com/Azure/ms-rest-js/pull/166/commits/46645f42238ef9e2240d7848640a294f7d324f5a.

**Pull request merged**